libwallet/examples/account_generation.rs

```diff
@@ -15,10 +15,10 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     };
 
     let mut wallet = Wallet::new(vault);
-    wallet.unlock(None).await?;
-    let account = wallet.default_account();
+    wallet.unlock(None, None).await?;
+    let account = wallet.default_signer();
 
     println!("Secret phrase: \"{phrase}\"");
-    println!("Default Account: 0x{account}");
+    // println!("Default Account: 0x{account}");
```

Comment:
If we are still returning a `default_account()` with `.public()` we can show the account ;)